org.torproject.torbrowser.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.torproject.torbrowser
+
+runtime: org.gnome.Platform
+runtime-version: "3.32"
+sdk: org.gnome.Sdk
+
+command: launch.sh
+
+finish-args:
+  - --socket=x11
+  # - --share=ipc
+  - --share=network
+
+modules:
+  - name: torbrowser
+    buildsystem: simple
+    build-commands:
+        - install -D Browser/browser/chrome/icons/default/default64.png   /app/share/icons/hicolor/64x64/apps/org.torproject.torbrowser.png
+        #- install -D Browser/start-tor-browser.desktop /app/share/applications/org.torproject.browser.desktop
+        - sed -i s,Icon=torbrowser,Icon=org.torproject.torbrowser,  torbrowser.desktop
+        - sed -i "s,Exec=torbrowser-launcher %u,Exec=/app/bin/launch.sh,"  torbrowser.desktop
+        - install -D torbrowser.appdata.xml  /app/share/metainfo/org.torproject.torbrowser.appdata.xml
+        - install -D torbrowser.desktop  /app/share/applications/org.torproject.torbrowser.desktop
+        - mv Browser /app
+        - install -D -m a=rx launch.sh   /app/bin/launch.sh
+    
+    cleanup:
+      - /Browser/TorBrowser/Docs/
+    sources:
+        - type: file
+          url: https://raw.githubusercontent.com/micahflee/torbrowser-launcher/develop/share/metainfo/torbrowser.appdata.xml
```

Comment:
This is used instead of the _org.torproject.torbrowser.appdata.xml_ file included in this PR?

Reply:
ah, yeah, good catch. I thought that appdata file was good enough, but it is indeed lacking. So it should be removed, I guess.